src/main/kotlin/org/jetbrains/research/testspark/helpers/psiHelpers/JavaPsiHelper.kt

```diff
@@ -111,7 +112,7 @@ class JavaPsiHelper : PsiHelperInterface {
         val psiMethod: PsiMethod? = getSurroundingMethod(psiFile, caret.offset)
         val line: Int? = getSurroundingLine(psiFile, caret.offset)?.plus(1)
 
-        result.add(getClassDisplayName(psiClass))
+        result.add(ClassTypeProcessHelper.getClassDisplayName(psiClass))
         psiMethod?.let { result.add(getMethodDisplayName(it)) }
```

Comment:
If we were to have the `getClassDisplayName` as an extension method the call would look very pretty and neat: `psiClass.getDispalyName()`.